tracing/README.md

```diff
@@ -0,0 +1,37 @@
+# Tracing package
+
+`package tracing` provides an API wrapper around the tracing library
+[jaeger](https://github.com/uber/jaeger-client-go) that allows you to
+instrument operations in your application.
```

Comment:
The phrase "allows you to instrument operations" seems odd to me. Would it be better to say "record" or "trace"? Or "see instrumentation data from your application"?